content/governance.md

```diff
@@ -0,0 +1,175 @@
+
+## Decision making
+
+### Team members 
+
+Team member status may be given to those who have made ongoing contributions for at least 3 months to the Prometheus Project. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account. 
+
+New members may be proposed by any existing member by email to [prometheus-team][team]. It is highly desired to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote).
+
+Team members are added to the [GitHub organization][gh] as _Owner_. They should however respect the maintainers of each project.
+
+Team members may retire at any time by emailing [the team][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+Unless voted upon otherwise, membership changes take effect immediately.
+
+The current team members are:
+
+* Ben Kochie
+* Björn Rabenstein
+* Brian Brazil
+* Conor Broderick
+* Fabian Reinartz
+* Frederic Branczyk
+* Goutham Veeramachaneni
+* Johannes Ziemke
+* Julius Volz
+* Matt Layher
+* Matthias Rampke
+* Max Inden
+* Richard Hartmann
+* Steve Durrheimer
+* Stuart Nelson
+* Tobias Schmidt
+* Tom Wilkie
+
+### Maintainers
+
+Maintainers lead one or more project(s) or parts thereof and serve as a point of conflict resolution amongst the contributors to this project. Ideally, maintainers are also team members, but exceptions are possible for suitable maintainers that, for whatever reason, are not or not yet team members.
+
+Changes in maintainership have to be announced on the [developers mailing list][devs]. They are decided by [lazy consensus](#consensus) and formalized by changing the `MAINTAINERS.md` file of the respective repository.
+
+Maintainers are granted commit rights to all projects in the [GitHub organization][gh].
+
+A maintainer or committer may resign by notifying the [team mailing list][team]. A maintainer with no project activity for a year is considered to have resigned. Maintainers who wish to resign are encouraged to propose another team member to take over the project.
+
+A project may have multiple maintainers, as long as the responsibilities are clearly agreed upon between them. This includes coordinating who handles which issues and pull requests.
+
+Any maintainership change is performed by changing the project's MAINTAINERS.md on the main branch of the project.
```

Comment:
We should consistently either use MAINTAINERS.md or `MAINTAINERS.md`. (I prefer the latter.)

Reply:
In different news, this information is already contained in line 69, so we don't need this line anymore.

Reply:
removed it, and changed all the file references to be in `MAINTAINERS.md` style.